Dev (#47)
42-AI/bootcamp_data-engineering: day01/ex10/ex10.md

# Exercise 10 - Kibana

|                         |                    |
| -----------------------:| ------------------ |

> |   Turnin directory :    |  ex10              |

Reviewer: ```suggestion
|   Turn-in directory:    |  ex10              |

```

> |   Files to turn in :    |  ex10-rendu.txt    |

Reviewer: ```suggestion
|   Files to turn in:     |  ex10-rendu.txt    |

```

> |   Forbidden function :  |  None              |

Reviewer: ```suggestion
|   Forbidden functions:  |  None              |

```

> |   Remarks :             |                    |

Reviewer: ```suggestion
|   Remarks:              |                    |

```

> We will now explore a little Kibana.

Reviewer: ```suggestion
Time to explore Kibana a little bit more.

```

> - Your goal is to create a Dashboard with the following visualization :

Reviewer: ```suggestion
- Your goal is to create a Dashboard with the following visualizations:

```

  - A plot showing the number of games released (Y axis) over time (X axis)
  - A histogram that counts the number of games released each year, and for each year the count of the "average user rating" by an interval of 1
  - A Pie Chart showing the repartition of Genres
  - A cloud of words showing the top developers

> - Once your dashboard is created, explore the possibilities of Kibana (click on the top developer in the cloud of words for instance).

Reviewer: ```suggestion
- Once your dashboard has been created, explore the possibilities of Kibana (click on the top developer in the cloud of words for instance).

```

> Your Dashboard should look something like this :  

Reviewer: ```suggestion
Your Dashboard should look like something like this:  

```
